spec/requests/citizens/transactions_spec.rb

```diff
@@ -106,7 +106,9 @@
 
       it 'redirects to the next page' do
         subject
-        expect(response.body).to include('Summary of income or payments')
+        expect(response).to redirect_to citizens_income_summary_path
+        follow_redirect!
+        expect(response.body).to include('Select your income')
```

Comment:
The follow is unnecessary and now makes this spec more likely to fail on markup changes to the next page.

Reply:
Not sure I agree.  the idea of a request spec is that it tests behaviour, not implementation.  So actually, it would be better to remove the `expect(response).to redirect_to citizens_income_summary_path` and keep the `follow_redirect!` and the expectation on the page which is displayed.